determine_needed_tags.sh

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env bash
+
+# the first argument must be the name of the container, i.e. st2, st2actionrunner, st2stream
+component=$1
+# the 2nd argument is the version of the current build and expects at least major.miinor to be provided
+# i.e. 3.3.0, 3.3, 2.10.5
+build_version=$2
+
+showHelp() {
+  echo "Use this script to determine the right tags to be attached to your new image."
+  echo
+  echo "Syntax: $0 st2component image_version"
+  echo "st2component = an valid image of an existing stackstorm component (i.e. st2, st2api, st2stream)"
+  echo "image_version = the st2 release used for your current build (i.e. 3.3.0)"
+  echo 
+  echo "Possible output:"
+  echo "0 = no additional tags to be updated"
+  echo "1 = update the major.minor tag (i.e. 3.3)"
+  echo "2 = update the major and the major.minor tag (i.e. 3 and 3.3)"
+  echo
```

Comment:
Missing description for the `3` option with the `latest` tag.

Reply:
Also done. Thanks for the review and feedback!